### PR TITLE
fix makefile to override image name

### PR DESCRIPTION
**Pull request by upodroid**
#155 didn't work as the makefile is ignoring the env override.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,9 @@ can make it available elsewhere if appropriate:
 sudo install ~/go/bin/cloud-provider-kind /usr/local/bin
 ```
 
+Starting with v0.4.0, the docker image for cloud-provider-kind is available
+at `registry.k8s.io/cloud-provider-kind/cloud-controller-manager:v0.4.0`
```
Comment (stmcginnis) [inline README.md:33]: I wonder if we should leave off the tag, or default it to `latest`. We'll either have to remember to update the README for every release, or we'll have a risk of people copy/pasting this without thinking and being stuck using what will eventually be an old release.

**Review by stmcginnis — COMMENTED** [review]
Good catch!

/lgtm

File: README.md (hunk 1 above)
Comment (upodroid) [reply README.md]: Its fixed, can you also edit this release as well? https://github.com/kubernetes-sigs/cloud-provider-kind/releases/tag/v0.4.0 the last line in the release note

**Comment by aojea** [comment]
/approve

Thanks

File: README.md (hunk 1 above)
Comment (aojea) [reply README.md]: done

**Comment by stmcginnis** [comment]
/lgtm

Thanks!

**Pull request merged**